libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -32,9 +32,9 @@ export class BookSearchComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.store.select(getAllBooks).subscribe(books => {
+   this.store.select(getAllBooks).subscribe(books => {
```

Comment:
Can you think of better way to handle this? It currently has a memory leak due to the subscription not being unsubscribed. Hint: See `ReadingListComponent` which doesn't have the same problem.

Reply:
@Jaysoo: This review has been addressed, please take a look. Thanks.